app/views/smart_proxies/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<% javascript('statuses') %>
+<% title(_('Smart Proxy: %s') % @smart_proxy.to_label) %>
+<%= smart_proxy_title_actions(@smart_proxy, authorizer) %>
+<div id="proxy-show" class="row" data-history-url='<%= smart_proxy_path(@smart_proxy)%>'>
+  <div class="col-md-4">
+    <div class="tab-content">
+      <table id="details_table" class="table table-bordered table-striped">
+        <thead>
+          <tr>
+            <th colspan="2"><%= _('Details') %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><%= _('Registration date') %></td>
+            <td><%= @smart_proxy.created_at.to_s(:long) %></td>
```

Comment:
This doesn't appear to be localised, it's rendering in a US style here, check it's using the user's TZ.